docs/modules/ROOT/pages/integrating-with-existing-deployments.adoc

```diff
@@ -43,15 +43,15 @@ export CLUSTER_MONITOR_PWD=password
 docker run -d --rm \
   -p 7196:7196 \ <.>
   --network cmos \
-  --couchbase-user Administrator
-  --couchbase-password password
+  --couchbase-user Administrator \
+  --couchbase-password password \
+  -e CB_MULTI_ALERTMANAGER_URLS=http://<Alertmanager IP>:9093 \
```

Comment:
Perhaps add a comment / callout that you can omit it if you're not using Alertmanager?